nevergrad/functions/images/core.py

```diff
@@ -12,10 +12,12 @@
 import os
 from nevergrad.common.typetools import ArrayLike
 from .. import base
+import torch.nn as nn
+import torch
 
 
 class Image(base.ExperimentFunction):
-    def __init__(self, problem_name: str = "recovering", index: int = 0) -> None:
+    def __init__(self, problem_name: str = "recovering", index: int = 0, params: dict = {}) -> None:
```

Comment:
Cool, the change of signature is worth it :-)

Reply:
you'll need to `import typing as tp`
then `params: tp.Optional[tp.Dict[str, tp,Any]] = None`
As a general rule, *never* set a dict or a list as a default argument, this can lead to reaaaaally unexpected behavior, I can show you how at some point :D 
Better initialize it as None, then
```python
if params is None:
    params = {}
```